README.md

```diff
@@ -65,6 +65,12 @@ This scenario leverages the MSBuildForUnity [Project Builder](#msbuild-project-b
 
 1. Add additional references to any NuGet packages you want to use in your Unity project.
 
+## Extended Instructions
+
+Here you can find additional instructions for various things you may want to accomplish:
+
+- [Add Support MSB4U to a NuGet Package](Documentation/CraftingNuGetPackages.md)
```

Comment:
nit `Add MSB4U Support to a NuGet Package`. Was MSB4U declared as an acronymn anywhere in a preceeding ? if not, i would continue to call things MSBuildForUnity even though its longer to detract from confusion. In the very first title you could have `MSBuildForUnity (MSB4U)`